cdap-docs/reference-manual/source/http-restful-api/monitor.rst

```diff
@@ -111,6 +130,100 @@ The status of these CDAP system services can be checked:
    * - Description
      - Returns the status of the metrics service
 
+Container Information of a CDAP System Service
+----------------------------------------------
+If you are trying to debug a system service, you can retrieve container info for a system service with::
```

Comment:
it may be worth mentioning that this is only for distributed cdap and not the sdk.